### ISSUE TITLE
Opening DevTools TreeView may crash gnome

### ISSUE THREAD

**Issue by Ramil2911** [AvaloniaUI/Avalonia#6108]
Pressing root in Logical Tree in Logical Tree crashes gnome,
![image](https://user-images.githubusercontent.com/19269321/122645151-5afc8280-d132-11eb-9c57-2faeaa01df88.png)
but if you open Visual Tree first, it works just fine
![image](https://user-images.githubusercontent.com/19269321/122645182-7ff0f580-d132-11eb-9429-246d45a277a0.png)
![image](https://user-images.githubusercontent.com/19269321/122645190-8aab8a80-d132-11eb-85cb-63b8d5570e14.png)

- OS: Fedora Linux 34 (Workstation Edition), gnome-shell 40.1-1.fc34.x86_64
- Version 0.10.6

**Comment by kekekeks** [comment]
Does it somehow crash the entire gnome-shell?

**Comment by Ramil2911** [comment]
> Does it somehow crash the entire gnome-shell?

Yes, gnome freezes and shows glitches after some time, you need to restart device or graphics to restore functionality.

**Comment by timunie** [comment]
closing this in favor of the new DevTools 
https://docs.avaloniaui.net/tools/developer-tools/installation
